kyo-data/shared/src/main/scala/kyo/Render.scala

```diff
@@ -16,63 +16,56 @@ sealed trait LowPriorityRenders:
 object Render extends LowPriorityRenders:
     inline def apply[A](using r: Render[A]): Render[A] = r
 
+    def instance[A](impl: A => Text): Render[A] =
```

Comment:
```suggestion
    def from[A](impl: A => Text): Render[A] =
```

I would like to avoid `instance` for non-singleton style values.